### PR TITLE
fix: MMASSETS-476 Localize ALT and provide TITLE for AvatarToken and AvatarNetwork

### PR DESCRIPTION
**Pull request by darkwing**
## **Description**

This PR serves two purposes:

1.  Adds a `title` attribute to the `AvatarNetwork` and `AvatarToken` components so a native tooltip displays the token or network name upon hover
2. Localizes some text that was not localized properly

[Open in GitHub Codespaces](https://codespaces.new/MetaMask/metamask-extension/pull/29215?quickstart=1)

## **Related issues**

Fixes:

## **Manual testing steps**

1. Go to the asset list
2. Hover over the network portion of a AvatarNetwork
3. See native tooltip with network name

## **Pre-merge author checklist**

- [ ] I've followed [MetaMask Contributor Docs](https://github.com/MetaMask/contributor-docs) and [MetaMask Extension Coding Standards](https://github.com/MetaMask/metamask-extension/blob/main/.github/guidelines/CODING_GUIDELINES.md).
- [ ] I've completed the PR template to the best of my ability
- [ ] I’ve included tests if applicable
- [ ] I’ve documented my code using [JSDoc](https://jsdoc.app/) format if applicable
- [ ] I’ve applied the right labels on the PR (see [labeling guidelines](https://github.com/MetaMask/metamask-extension/blob/main/.github/guidelines/LABELING_GUIDELINES.md)). Not required for external contributors.

## **Pre-merge reviewer checklist**

- [ ] I've manually tested the PR (e.g. pull and build branch, run the app, test code being changed).
- [ ] I confirm that this PR addresses all acceptance criteria described in the ticket it closes and includes the necessary testing evidence such as recordings and or screenshots.

### CODE REVIEW

File: ui/components/app/multi-rpc-edit-modal/network-list-item/__snapshots__/network-list-item.test.tsx.snap [hunk 1]
```diff
@@ -12,9 +12,10 @@ exports[`NetworkListItem renders correctly with required props 1`] = `
         class="mm-box mm-text mm-avatar-base mm-avatar-base--size-md mm-avatar-network mm-text--body-sm mm-text--text-transform-uppercase mm-box--display-flex mm-box--justify-content-center mm-box--align-items-center mm-box--color-text-default mm-box--background-color-background-alternative mm-box--rounded-md mm-box--border-color-background-default mm-box--border-width-1 box--border-style-solid"
       >
         <img
-          alt="Ethereum Mainnet logo"
+          alt="[avatarImageLogo]"
```
Comment (georgewrmarshall) [inline ui/components/app/multi-rpc-edit-modal/network-list-item/__snapshots__/network-list-item.test.tsx.snap:15]: Is this the correct alt?

File: ui/components/multichain/asset-picker-amount/asset-picker/__snapshots__/asset-picker.test.tsx.snap [hunk 2]
```diff
@@ -156,9 +158,10 @@ exports[`AssetPicker should render network picker when networks prop is defined
               class="mm-box mm-text mm-avatar-base mm-avatar-base--size-md mm-avatar-token mm-text--body-sm mm-text--text-transform-uppercase mm-box--display-flex mm-box--justify-content-center mm-box--align-items-center mm-box--color-text-default mm-box--background-color-background-alternative mm-box--rounded-full"
             >
               <img
-                alt="NATIVE TICKER logo"
+                alt="[avatarImageLogo]"
```
Comment (georgewrmarshall) [inline ui/components/multichain/asset-picker-amount/asset-picker/__snapshots__/asset-picker.test.tsx.snap:161]: Should this have an alt?

**Review by georgewrmarshall — COMMENTED** [review]
I love these updates but just curious if the alt tags should have the bracket notation or just be empty.

**Comment by darkwing** [comment]
Superceded by https://github.com/MetaMask/metamask-extension/pull/29336

**Pull request closed without merging**